README.md

```diff
@@ -1,155 +1,189 @@
-<!-- [![Build and Test](https://github.com/dhiway/cord/workflows/Build%20and%20Test/badge.svg)](https://github.com/dhiway/cord/actions) -->
+This repository contains the Rust implementation of a [Cord Network][cord-homepage] node based on the [Substrate][substrate-homepage] framework.
 
 # CORD
 
-The CORD node implementation uses Parity Substrate as the underlying
-technology stack with custome pallets and hierarchical trust Modules.
+Cord is designed to be a global public utility and trust framework that creates new possibilities in how we address trust gaps, manage transactions and exchanges of value at scale.
```

Comment:
CORD is a global public utility and trust framework that is designed to address trust gaps, manage transactions and exchange of value at scale.